src/components/gallery/HdGalleryCarousel.vue

```diff
@@ -150,7 +152,10 @@ export default {
     },
     updateCurrentIndex(itemIndex) {
       this.currentIndex = itemIndex;
-      if (window.matchMedia('min-width: $break-tablet')) {
+
+      const mediaQuery = `(max-width: ${sassVariables.breakTablet})`;
+
+      if (mediaMatches(mediaQuery)) {
```

Comment:
@aym3nb pin pointed the bug here, the problem is that we use the same instance of flickity on desktop and mobile. However flickity works differently on those devices (on desktop index is the number of the page, while on mobile, it's the index of the image active). So on desktop, we should not emit this event. 

@aym3nb , please add more context here, if needed.